test/integration2/schema/misc.spec.js

```diff
@@ -543,19 +551,19 @@ describe('Schema (misc)', () => {
         });
 
         describe('enum - postgres', () => {
-          if (!isPgBased(knex)) {
-            return Promise.resolve();
-          }
-
           afterEach(() =>
             knex.schema
               .dropTableIfExists('native_enum_test')
               .raw('DROP TYPE IF EXISTS "foo_type"')
               .raw('DROP SCHEMA IF EXISTS "test" CASCADE')
           );
```

Comment:
We'll have to add a PG check here too. There are a few tests breaking because of SQL syntax error.